launcher.py

```diff
@@ -36,8 +35,14 @@ def gen_launcher_command(settings):
 		'SYS.Region='+settings['region'],
 		'ver='+settings['version']]
 	#Deal with an empty wine_command (Running on windows)
-	if launcher_dict[0] == '':
+	if (launcher_dict[0] == '') or (settings['mac_app'] != ''):
 		launcher_dict = launcher_dict[1:]
+
+	if (settings['mac_app'] != ''):
+		cider_bin = join_path(settings['mac_app'], 'Contents/MacOS/FINALFANTASYXIV')
+		cmdline = ' '.join(launcher_dict)
+		launcher_dict = [cider_bin, '-cmdline', cmdline, exe_path]
```

Comment:
Correct. This is what Cider (the proprietary fork of Wine that FFXIV uses on Mac) requires.